doc/dev/developer_guide/essentials.rst

```diff
@@ -13,20 +13,19 @@ following table shows all the leads and their nicks on `GitHub`_:
 
 .. _github: https://github.com/
 
-========= ================ =============
-Scope     Lead             GitHub nick
-========= ================ =============
-Ceph      Sage Weil        liewegas
-RADOS     Neha Ojha        neha-ojha
-RGW       Yehuda Sadeh     yehudasa
-RGW       Matt Benjamin    mattbenjamin
-RBD       Ilya Dryomov     dis 
-CephFS    Venky Shankar    vshankar
-Dashboard Ernesto Puerta   epuertat
-MON       Joao Luis        jecluis
-Build/Ops Ken Dreyer       ktdreyer
-Docs      Zac Dover        zdover23
-========= ================ =============
+========= ================== =============
+Scope     Lead               GitHub nick
+========= ================== =============
+RADOS     Radoslaw Zarzynski rzarzynski 
+RGW       Casey Bodley       cbodley 
+RGW       Matt Benjamin      mattbenjamin
+RBD       Ilya Dryomov       dis 
+CephFS    Venky Shankar      vshankar
+Dashboard Nizamudeen A       nizamial09 
+MON       Joao Luis          jecluis
```

Comment:
TMK Joao is no longer working on Ceph or maintaining the monitors anymore.

Reply:
@batrick: Thanks, Patrick. I'll make a follow-up PR when I'm done with breakfast.

Reply:
https://github.com/ceph/ceph/pull/56600 - This PR removes Joao from the list of leads.